credits.txt

```diff
@@ -1,6 +1,14 @@
 Welcome to Endless Sky!
 version 0.9.12
 
+This WebAssembly port done by:
+  janisozaur
+  thomasballinger
+
+Fork available at
+  https://github.com
+    /janisozaur/endless-sky
```

Comment:
looks ugly, but wouldn't have fit in the fairly narrow box.

https://git.io/ provides a shorter version, but harder to retype from a scrolling text area (no way to select): https://git.io/JTw6x